configdependabotfile.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bufio"
+	"strings"
+
+	dependabot "github.com/paulvollmer/dependabot-config-go"
+	"gopkg.in/yaml.v2"
+)
+
+type configDependabotResponse struct {
+	OriginalInput        string
+	FinalOutput          string
+	IsChanged            bool
+	configfileFetchError bool
+}
+
+func configDependabot(configDependabotFile string) (*configDependabotResponse, error) {
```

Comment:
It should be called something like `UpdateDependabotConfig`. Method names should start with a verb that indicates the action to be performed, e.g. `PrintReport(`)